drivers/ethernet/eth_stellaris.c

```diff
@@ -0,0 +1,365 @@
+/* STELLARIS Ethernet Controller
+ *
+ * Copyright (c) 2018 Zilogic Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define SYS_LOG_DOMAIN "dev/eth_stellaris"
```

Comment:
While this was fixed, the syslog is being deprecated, so these two syslog settings need to be replaced by
```
#define LOG_MODULE_NAME eth_stellaris
#define LOG_LEVEL CONFIG_ETHERNET_LOG_LEVEL
#include <logging/log.h>
LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Reply:
This has been fixed.